### DISCUSSION TITLE
mbtiles

### DISCUSSION THREAD

**Discussion by prancius** [maptiler/tileserver-gl#979] [General]
If it possible to download europe mbtiles free?
i have try openmaptiles from https://osmlab.github.io/osm-qa-tiles/ but it not working

**Comment by utagawal** [comment]
Hello,
It used to be feasible on [https://osm.dbtc.link/mbtiles/](https://osm.dbtc.link/mbtiles/) but since a couple of months it is not updated.

**Comment by acalcutt** [comment]
I recently posted some full planet torrents on my own site at https://www.techidiots.net/downloads/open-base-map-torrents/ , you can download them using any torrent client like qbittorent, transmission, or utorrent 

There is a full planet OpenMapTiles generated by [planetiler ](https://github.com/onthegomap/planetiler) here
https://www.techidiots.net/downloads/open-base-map-torrents/mbtiles/mbtiles-vector-openmaptiles-openstreetmap-vector-tiles-generated-by-planetiler . 

Please be aware I have been experiencing some server issues today, so if you go to the site and it is down please try again later. I am trying to figure out the cause right now.

**Comment by acalcutt** [comment]
I strongly suggest planetiler https://github.com/onthegomap/planetiler for generating your own mbtiles. you can read their doc on generating planet tiles here https://github.com/onthegomap/planetiler/blob/main/PLANET.md

I can generate a full planetin 8.5hrs on a laptop with 32GB memory with 
```
java -Xmx24g \
  -jar planetiler.jar \
  --area=planet --bounds=planet --download \
  --download-threads=10 --download-chunk-size-mb=1000 \
  --fetch-wikidata \
  --output=output.mbtiles \
  --nodemap-type=array --storage=mmap
```
  
  you can also make one just for Europe by setting '--area=planet' to '--area=europe' , which would be a lot faster and use less space.

**Comment by utagawal** [comment]
If you do not have the HW available to run planetiler, you can try [https://github.com/lambdajack/sequentially-generate-planet-mbtiles](https://github.com/lambdajack/sequentially-generate-planet-mbtiles)

**Comment by prancius** [comment]
nice to know, thank you. I will try
